packages/cms-cli/hubspot.sample.config.yml

```diff
@@ -32,3 +32,13 @@ portals:
         refreshToken: 84d22710-4cb7-5581-ba05-35f9945e5e8e
         accessToken: >-
           CJDVnLanLRICEQIYyLu8LyDh9E4opf1GMhkAxGuU5XN_O2O2QhX0khw7cwIkPkBRHye-OfIADgLBAAADAIADAAAAAAAAAAJCGQC8a5TlhtSU8T-2mVLxOBpxS18aM42oGKk
+  - name: QA
```

Comment:
I don't think we want to add another entry into the sample config. When we're ready to switch over to usertoken being the default, we can update the sample config and only user `usertoken` auth types.